src/pages/media.tsx

```diff
@@ -1,36 +1,39 @@
 import React from "react";
 import { Helmet } from "react-helmet";
-import { ResourceEvent } from "../components/UI/ResourceEvent";
+import { ResourceEvent, ResourceEventProps } from "../components/UI/ResourceEvent";
 import { ResourceLink } from "../components/UI/ResourceLink";
 import { Resource } from "../types";
 
 const pressReleases: Resource[] = [
   {
-    title:
-      "Launch of TradeTrust announced by Mr S Iswaran, Minister for Communications and Information at the Committee of Supply Debate on 4 March 2019",
-    date: "4 Mar 2019",
+    title: "Adoption of UNCITRAL model law heralds a quiet revolution in digital trade",
+    date: "3 Feb 21",
     url:
-      "https://www.mci.gov.sg/pressroom/news-and-stories/pressroom/2019/3/securing-our-future-in-a-digital-age#.X5gFH3jrCCc.gmail",
+      "https://www.businesstimes.com.sg/opinion/adoption-of-uncitral-model-law-heralds-a-quiet-revolution-in-digital-trade ",
+  },
+  {
+    title: "Singapore leading the way in promoting digital trade",
+    date: "3 Feb 21",
+    url: "https://www.businesstimes.com.sg/opinion/singapore-leading-the-way-in-promoting-digital-trade ",
   },
   {
     title:
-      "World's Largest Business Organisation Joins Singapore Government and Major Industry Partners to Drive Global Digitalisation",
-    date: "22 Jan 2020",
+      "Electronic Transactions Act Amended To Facilitate Electronic Transactions, Providing Convenience And Strengthening Singapore’s Trade Competitiveness In The Digital Economy",
+    date: "1 Feb 21",
```

Comment:
date convention varies from resources
here its 1 Feb 21
below is 5 Oct 2020

Reply:
updated to follow D MMM YYYY for now